docs/our_work/Papi_Webpage.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Risk stratification models for Population and Person Insights (PaPI)'
+summary: 'To improve, and update the risk stratification models used in the Population and Person Insight (PaPI) dashboards.'
+origin: 'NHS England'
+tags: ['PAPI', 'PUBLIC', 'POPULATION HEALTH', 'MACHINE LEARNING', 'MODELLING', 'STRUCTURED', 'PYTHON']
+---
```

Comment:
try and only use the tags in the tables at the top: https://nhsengland.github.io/datascience/our_work/tags/ so both modelling, public, and papi would be excess here and papi specially is a bit too niche. If you could pick one tag from each of the categories at the top that would be great!